src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -65,4 +85,16 @@ public function getBlockPrefix(): string
     {
         return 'country';
     }
+
+    private static function getEmojiFlag(string $name): string
+    {
+        return self::toRegionalSymbol($name, 0) . self::toRegionalSymbol($name, 1);
```

Comment:
Does this logic works fine when using alpha3 country codes ?

Reply:
Yes the second screen in description

Reply:
In this second screenshot, it seems that the list of countries is different from the one in the first screenshot, is it intentional?